project.clj

```diff
@@ -9,9 +9,9 @@
    :password :env/nexus_jenkins_password
    :sign-releases false})
 
-(def tk-version "1.2.0")
+(def tk-version "1.3.1")
```

Comment:
I'd hold off. I think there's enough variables here, seems useful to test this on it's own before another version bump. Also we need to make sure the other PE projects are bumping to 1.4.0 for the next release before we bump.